pom.xml

```diff
@@ -577,6 +577,25 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.18.1</version>
 			</plugin>
+			<plugin>
```

Comment:
is this change part of this PR ??

Reply:
it is not, it's an issue in my development environment, i wanted to push it for another branch, ignore this commit

Reply:
I reverted this commit, kindly resolve this conversation